mono/mini/mini-runtime.h

```diff
@@ -422,10 +422,11 @@ MONO_API void        mono_parse_env_options         (int *ref_argc, char **ref_a
 MONO_API char       *mono_parse_options_from        (const char *options, int *ref_argc, char **ref_argv []);
 MONO_API int         mono_regression_test_step      (int verbose_level, const char *image, const char *method_name);
 
-
```

Comment:
Unrelated whitespace change. Generally avoid these in diffs.

Reply:
This is still not resolved. You should check the actual diff before marking anything.